### ISSUE TITLE
Add EMELD???

### ISSUE THREAD

**Issue by shooyenga** [HughP/LinguisticsLibrarianList#6]
The E-MELD site (http://emeld.org/, hosted by Linguist List) hasn't been updated for years, but might still have some useful information, in case you haven't seen it.

**Comment by HughP** [comment]
Good idea, I see how E-MELD is relevant to archivists, is there other ways which it might also be relevant to librarians?
